templates/kubeflow/what-is-kubeflow.html

```diff
@@ -187,80 +121,294 @@ <h2 class="p-muted-heading u-align--center">Contributors to Kubeflow</h2>
+        </li>
+        <li class="p-inline-images__item ">
+        {{
+          image(
+            url="https://assets.ubuntu.com/v1/614f1f45-Pachyderm_logo.svg",
+            alt="Pachyderm",
+            height="120",
+            width="130",
+            hi_def=True,
+            attrs={"class": "p-inline-images__logos"},
+            loading="lazy",
+          ) | safe
+        }}
+        </li>
+      </ul>     
     </div>
-    <div class="col-6 p-card">
-      <h3 class="p-card__thumbnail">Serving</h3>
-      <hr>
-      <div class="p-card__description">
-        <p>Kubeflow makes two service systems available, KFServing and Seldon Core. These allow multi-framework model serving and the choice should be made based on the needs of each project.</p>
+  </div>
+</section>
+
+<section class="p-strip--square-darksuru is-dark">
+  <div class="row u-equal-height">
+    <div class="col-7 u-vertically-center">
+      <div>
+        <h2>Kubeflow, well packaged</h2>
+        <p>Charmed Kubeflow integrates the <a href="https://jaas.ai/u/kubeflow-charmers#charms" class="p-link--external p-link--inverted">30+ apps</a> that make up Kubeflow with ops code to provide the best Kubeflow experience, from deployment to day-2 operations.</p>
+        <br />
       </div>
     </div>
-    <div class="col-6 p-card">
-      <h3 class="p-card__thumbnail">Training</h3>
-      <hr>
-      <div class="p-card__description">
-        <p>Model training is possible with various frameworks, in particular TensorFlow, Chainer, MPI, MXNet and PyTorch. These cover the most popular frameworks in the data science and AI/ML space ensuring that developers are able to use Kubeflow’s MLOps features with their favourite tools.</p>
+    <div class="col-5 u-align--center">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/2b8bb6e7-Kubeflow-packaged-transparent.svg",
+          alt="",
+          width="195",
+          height="180",
+          hi_def=True,
+          attrs={"class": "u-hide--small"},
+          loading="lazy",
+        ) | safe
+      }}
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width u-sv1">
+    <h2>Why MLOps?</h2>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p>Bringing AI solutions to market can involve many steps: data pre-processing, training, model deployment or inference serving at scale... The list of tasks is complex and keeping them in a set of notebooks or scripts is hard to maintain, share and collaborate on, leading to inefficient processes.</p>
+      <p>
+      In <a class="p-link--external" href="https://papers.nips.cc/paper/5656-hidden-technical-debt-in-machine-learning-systems.pdf">[1]</a>, Google describes that only about 20% of the effort and code required to bring AI systems to production is the development of ML code, while the remaining is operations. Standardizing ops in your ML workflows can hence greatly decrease time-to-market and costs for your AI solutions.
```

Comment:
This is in the copy doc, but I don't think "[1]" gives the user much indication what is being linked to - maybe it's a convention I'm not aware of... @pmahnke? Anyway, would suggest changing it to:
```suggestion
      In <a class="p-link--external" href="https://papers.nips.cc/paper/5656-hidden-technical-debt-in-machine-learning-systems.pdf">this study</a>, Google describes that only about 20% of the effort and code required to bring AI systems to production is the development of ML code, while the remaining is operations. Standardizing ops in your ML workflows can hence greatly decrease time-to-market and costs for your AI solutions.
```

Reply:
@bethcollins92 I think this is better